### DISCUSSION TITLE
Add milliwatts as unit of power

### DISCUSSION THREAD

**Discussion by agners** [home-assistant/architecture#1174] [Entity Models] [answered]
Home Assistant currently has W (watts), kW (kilowatts) MW (megawatts) GW (gigawatts) and TW (terawatts) as units of power for electrical power.

Since Matter v1.3, home appliances using the protocol can report their power usage using the Electrical Power Measurement cluster. The power measurement cluster reports power in mW (milliwatts).

## Proposal

Add the milliwatt equivalent as unit of power:

- `mW` (milliwatts) as unit of power

Implementation for [mW as unit of power](https://github.com/home-assistant/core/pull/133504) is already present.

## Integrations which can benefit from the proposal

Integrations that can benefit are primarily for devices with accurate electrical power measurement:

- Matter
- Enphase Envoy solar energy monitor

**Comment by frenck** [comment, marked as answer]
We have discussed this proposal and are ok with adding it 👍 

../Frenck
